### DISCUSSION TITLE
Can't use wallpapers

### DISCUSSION THREAD

**Discussion by sekutton** [Taiizor/Sucrose#75] [Q&A] [answered]
Hi Taiizor!
I recently downloaded your application, but turns out I can't use, preview, personalize, nor remove any of the wallpapers downloaded through the gallery. How can I fix that?

**Comment by Taiizor** [comment]
Hello,

Preview and Customization are currently not in use.

May i inquire about which version of Sucrose you are using for other issues?

**Reply by sekutton** [reply]
What do you mean when you say there are not in use?

Also I have the 24.1.2.0 version

**Reply by sekutton** [reply]
The problem is that I cannot set the wallpapers. The button is grey-ish, and I cannot click it :/

**Reply by Taiizor** [reply]
It is currently not in use as it is still in the development stage (Preview & Customization).

You are using one of the latest versions. The reason you cannot change the wallpaper may be that any condition in the Performance settings has been met. Performing such actions is prevented when conditions are met.

https://github.com/Taiizor/Sucrose/assets/41683699/21cd23ba-0720-437f-9a4b-7881993bda6d

**Reply by Taiizor** [reply, marked as answer]
I provided an example with Memory usage in the video, but there might be a different performance setting condition being met on your end.

**Reply by sekutton** [reply]
Okay, I put most of the performance settings on "resume" and now it works perfectly! Thank you!

**Reply by Taiizor** [reply]
I have made the necessary adjustments regarding this. It has become more informative compared to the previous version.

![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/e00bf905-b22f-41a9-b642-4da36eb60b86)
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/213da23c-5930-4c84-b9f7-73076aabc357)

![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/34cf31ef-4ec5-4944-846d-a9a96cf4731b)
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/3bdd51f9-9f3b-46a3-bc16-862264084fdd)
